go/libkb/db.go

```diff
@@ -21,6 +21,7 @@ const (
 	DBTeamChain         = 0x10
 	DBUserPlusAllKeysV1 = 0x19
 
+	DBIncomingSharePreference        = 0xa4
```

Comment:
can you addthis to `IsPermDbKey` below/

Reply:
Ahh good call. I actuallly saw that first and thought this needed to be there but forgot. Thanks